### ISSUE TITLE
Browser javascript variables sent to admins

### ISSUE THREAD

**Issue by capiscuas** [idoco/intergram#41]
Hi, we love your bot and we are starting to testing it.

We added the widget in a Redmine instance, where sometimes the user asking for help is logged in, and we can get this 'username' from the javascript code loading the widget itself... so I was wondering if you could add some param that the widget.js could use to send as internal message(before the first user msg) to the admins on Telegram, so that they can know more info.

Other ideas to profit this parameter:
Get the user browser info / location / username  / current url, etc.

**Comment by DNNhost** [comment]
I agree,   Just installed the app on a website, the app is put into the skin, so its active on all pages. 
I would really like to know which url they are calling from 
If they are :
a logged in user to the website or maybe
logged into facebook or
google - twitter - Live - LinkedIn
so I would get some info about the person for the support task
Like intergram and Telegram a lot (new to it) but have to skim the market for an app
that can provide any of the mentioned data.
